build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidCompose.kt

```diff
@@ -38,5 +38,33 @@ internal fun Project.configureAndroidCompose(
         composeOptions {
             kotlinCompilerExtensionVersion = libs.findVersion("androidxCompose").get().toString()
         }
+
+        kotlinOptions {
+            freeCompilerArgs = freeCompilerArgs + buildComposeMetricsParameters()
+        }
+    }
+}
+
+private fun Project.buildComposeMetricsParameters(): List<String> {
+    val metricParameters = mutableListOf<String>()
+    val enableMetricsProvider = project.providers.gradleProperty("enableComposeCompilerMetrics")
+    val enableMetrics = (enableMetricsProvider.orNull == "true")
+    if (enableMetrics) {
+        val metricsFolder = File(project.buildDir, "compose-metrics")
```

Comment:
I don't think using File is a good idea, but I'm trying to use File to take into account Windows paths and such. There may be a better API, but I'm using this one because it only affects me when I add options and has little impact.

Reply:
Yeah, `File` here looks good for those reasons.